<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QWidget" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget_left" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>55</width>
     <height>720</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_userImage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonChat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonAddressBook">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonCollect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonFriends">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonLook">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSearch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonSettings">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>670</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonPhone">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButtonProgram">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>55</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>70</y>
     <width>885</width>
     <height>650</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageChat">
    <widget class="QWidget" name="widgetChat" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>250</width>
       <height>650</height>
      </rect>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>650</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widgetToolBar" native="true">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>450</y>
       <width>635</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QPushButton" name="pushButtonExpression">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonFile">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonShot">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRecords">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>15</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidgetShow">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>0</y>
       <width>630</width>
       <height>450</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageAddressBook">
    <widget class="QListView" name="listView_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>341</width>
       <height>831</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageCollect"/>
   <widget class="QWidget" name="pageFile"/>
   <widget class="QWidget" name="pageFriends"/>
   <widget class="QWidget" name="pageLook"/>
   <widget class="QWidget" name="pageSearch"/>
   <widget class="QWidget" name="pagePhone"/>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>0</y>
     <width>885</width>
     <height>70</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>855</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMax">
    <property name="geometry">
     <rect>
      <x>825</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMin">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_search" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>70</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEditSearch">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>25</y>
       <width>165</width>
       <height>24</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>搜索</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>25</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSearchIcon">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>25</y>
       <width>20</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_uid">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonTop">
    <property name="geometry">
     <rect>
      <x>765</x>
      <y>0</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>30</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
